promptsource/templates/amazon_reviews_multi/en/templates.yaml

```
@@ -44,42 +47,86 @@ templates:
 
       {{review_body}} |||
 
-      {{stars}}'
+      {{answer_choices[stars-1]}}'
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
-      original_task: null
-    name: Template_4
+      choices_in_prompt: false
+      metrics:
+      - Accuracy
+      - Other
+      original_task: true
+    name: prompt_review_to_star
     reference: Rating based on review body
+  37806754-58f7-4383-961a-fe2c88109fcd: !Template
+    answer_choices: 1|||2|||3|||4|||5
+    id: 37806754-58f7-4383-961a-fe2c88109fcd
+    jinja: 'Rate the product by the number of stars based on the review below: (1
+      being the lowest and 5 the highest)
+
+      ===
+
+      {{review_title}}. {{review_body}} |||
+
+      {{answer_choices[stars-1]}}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - Accuracy
+      - Other
+      original_task: true
+    name: prompt_body_title_to_star
+    reference: Rating based on review body,title
   7ecaf718-c85d-47f4-83cb-f14c58f2911f: !Template
     answer_choices: null
     id: 7ecaf718-c85d-47f4-83cb-f14c58f2911f
-    jinja: 'Guess the product category for which the below review is:
+    jinja: 'Guess the product category from the following review:
 
       ===
 
       {{review_body}} |||
 
       {{product_category}}'
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
-      original_task: null
-    name: Template_1
+      choices_in_prompt: false
+      metrics:
+      - Accuracy
+      - BLEU
```

```suggestion
      - Other
```
BLEU is so weird for unigrams... removing that
also for the record, accuracy might be non optimal either (there are 5'000 categories in the test set alone)